### DISCUSSION TITLE
How to contribute in the docs repo

### DISCUSSION THREAD

**Discussion by nirban256** [WebXDAO/start-here#484] [Q&A] [answered]
My question is

What will be the content of the docs repo for WebXDAO open source projects and what are the possible ways by which we can contribute to the docs repo?

Please give me answer because I am interested to contribute to this repo.

**Comment by shuklaritvik06** [comment, marked as answer]
https://github.com/WebXDAO/blockchain-dev-path

Hello,

Thank you for your interest in contributing to our docs repo! The above is our GitHub repo, built with Docusaurus. Url: https://devpath.webxdao.xyz/. You can explore the existing content and add more training courses, tutorials, and other helpful resources for developers.

Your contributions will be invaluable in helping us build a comprehensive resource for the community.

Looking forward to seeing your contributions!

Best regards,
Ritvik Shukla

**Comment by nirban256** [comment]
Thanks for the info @shuklaritvik06
